.reek

```diff
@@ -28,6 +28,7 @@ UnusedPrivateMethod:
     - ActiveJob::Logging::LogSubscriber
     - Users::PhoneConfirmationController
 UtilityFunction:
+  public_methods_only: true
```

Comment:
I've run into this problem report in multiple PRs since we enabled reek, and I had previously found creative ways around it. The issue is something like this:

```ruby
class Foo

  def build
    attrs = {}
    add_bar(attrs) if something?
  end

  def add_bar(attrs)
    attrs[:bar] = 123
  end
end
```

I.e. if you have a method in a class that acts on an object other than `self`. If the class is a service (e.g. `AttributeAsserter`) then arguably all the methods are acting on objects other than `self`.

Try running `reek app/controllers` and see what I mean.